src/platform_impl/macos/app_state.rs

```diff
@@ -457,3 +464,18 @@ unsafe fn window_activation_hack(ns_app: id) {
         }
     }
 }
+fn apply_activation_policy(app_delegate: &Object) {
+    unsafe {
+        use cocoa::appkit::NSApplicationActivationPolicy::*;
+        let ns_app = NSApp();
+        // We need to delay setting the activation policy and activating the app
+        // until we have the main menu all set up. Otherwise the menu won't be interactable.
+        let act_pol = get_aux_state_mut(app_delegate).activation_policy;
+        ns_app.setActivationPolicy_(match act_pol {
+            ActivationPolicy::Regular => NSApplicationActivationPolicyRegular,
+            ActivationPolicy::Accessory => NSApplicationActivationPolicyAccessory,
+            ActivationPolicy::Prohibited => NSApplicationActivationPolicyProhibited,
+        });
+        let () = msg_send![ns_app, activateIgnoringOtherApps: YES];
```

Comment:
This is already called in `AppState::launched` - and should first be called after `window_activation_hack` (I would think that yields better behaviour, though maybe it doesn't make a difference)

Reply:
You are correct, I accidentally left it there.